src/development/platform-integration/platform-channels.md

```diff
@@ -822,7 +822,12 @@ be invoked on the platform's main thread. When invoking channels in Flutter
 destined for the platform side, they need to be invoked on the root Isolate. The
 platform side's handlers can execute on the platform's main thread or they can
 execute on a background thread if a Task Queue is used. The result of the
-platform side handlers can be invoked asynchronously and on any thread.
+platform side handlers can be invoked asynchronously and on any thread when the
+Task Queue API is available, otherwise they must be invoked on the platform
+thread.
+
+{{site.alert.note}}The Task Queue API is available for Android on the `stable`
+channel. For iOS, it is currently available on the `master` channel.
```

Comment:
```suggestion
{{site.alert.note}}
  The Task Queue API is available for Android on the `stable`
  channel. For iOS, it is currently available on the `master` channel.
{{site.alert.end}}
```

Reply:
Did they change the requirement for the end tag at some point?  I swore these were rendering correctly previously.